src/cli.js

```diff
@@ -87,4 +102,24 @@ function printProjectsOnConsole(projects, numberOfProjectsPerResult, outputType)
 
 function readYamlFile(yamlFile) {
 	return yaml.safeLoad(fs.readFileSync(yamlFile, 'utf8'));
+}
+
+function getWebhookConfig(config, repoName) {
```

Comment:
This function will make better sense in migrate.js. Since cli.js is just providing a wrapper to integrate with command line operations, but actually logic is with migrate.
The logic for default value must not be re-written if we decide to use something else of cli.js